lib/KLabeledIcon.vue

```diff
@@ -29,11 +36,21 @@
       KIcon,
     },
     props: {
-      // If provided, will render a KIcon with the same 'icon' prop
+      // If provided, will render a KIcon using the value given as the `icon` prop. The icon will be prepended to the label.
       icon: {
         type: String,
         required: false,
       },
+      // If provided, will render a KICon using the value given as the `icon` prop. The iconAfter will be appended to the label.
+      iconAfter: {
+        type: String,
+        required: false,
+      },
+      // If provided, determines the color of the label and any icons that are provided.
```

Comment:
Following writing convention: Skip periods and unnecessary punctuation

https://kolibri-design-system.netlify.app/writing/#punctuation

```suggestion
      // If provided, determines the color of the label and any icons that are provided
```